build_modules/lib/src/module_library.dart

```diff
@@ -116,21 +121,25 @@ class ModuleLibrary {
         deps.add(linkedId);
       }
     }
+    // Allow two or fewer arguments so that entrypoints intended for use with
```

Comment:
Is this comment in the right place? Should it be on `_hasMainMethod` instead?

Reply:
whoops ya, refactor screwup